src/drive/styles/table.styl

```diff
@@ -154,9 +154,23 @@ mime-types('fil-file-', '../assets/icons/')
 
 .fil-file-filename
     flex 0 0 1.3rem
-    overflow hidden
-    text-overflow ellipsis
-    white-space nowrap
+
+.fil-file-filename-wrapper
+    display table
+    width   100%
+    border-collapse collapse
+
+    .fil-file-filename-and-ext
+        display table-cell
+        max-width 1px
+        width 100%
```

Comment:
`max-width 1px` and `width 100%`? What is this about? 😁 
I'll check out the PR, but on the surface it looks like something that should be done with flexbox.

Reply:
`table-cell` is the new `grid-layout` which is the new `flexbox` :trollface:

Reply:
After testing out the code, it *works*, but I'm really not confident in this? There's no reason for using table layout and all it quicks here. `.fil-file-filename-spinner` doesn't need to be in the DOM unless the spinner is displayed either.
Unless I'm missing a reason to use table layout, I really think this should be changed.